Neos.ContentGraph.DoctrineDbalAdapter/src/ContentGraphFactory.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Neos\ContentGraph\DoctrineDbalAdapter;
+
+use Neos\ContentGraph\DoctrineDbalAdapter\Domain\Repository\ContentGraph;
+use Neos\ContentGraph\DoctrineDbalAdapter\Domain\Repository\NodeFactory;
+use Neos\ContentRepository\Core\ContentGraphFactoryInterface;
+use Neos\ContentRepository\Core\Infrastructure\DbalClientInterface;
+use Neos\ContentRepository\Core\NodeType\NodeTypeManager;
+use Neos\ContentRepository\Core\Projection\Workspace\Workspace;
+use Neos\ContentRepository\Core\Projection\Workspace\WorkspaceStatus;
+use Neos\ContentRepository\Core\SharedModel\ContentRepository\ContentRepositoryId;
+use Neos\ContentRepository\Core\SharedModel\Exception\ContentStreamDoesNotExistYet;
+use Neos\ContentRepository\Core\SharedModel\Workspace\ContentStreamId;
+use Neos\ContentRepository\Core\SharedModel\Workspace\WorkspaceDescription;
+use Neos\ContentRepository\Core\SharedModel\Workspace\WorkspaceName;
+use Neos\ContentRepository\Core\SharedModel\Workspace\WorkspaceTitle;
+
+/**
+ * @internal only used inside the
+ * @see ContentGraphFinder
+ */
+readonly final class ContentGraphFactory implements ContentGraphFactoryInterface
```

Comment:
I wonder that the linters didn't seem to complain, but this is the order we use everywhere IIRC
```suggestion
final readonly class ContentGraphFactory implements ContentGraphFactoryInterface
```